draft-vvv-tls-alps.md

```diff
@@ -320,8 +331,20 @@ carried over from the previous connection.
 # IANA Considerations
 
 IANA will update the "TLS ExtensionType Values" registry to include
-`application_settings` with the value of TBD; the list of messages in which
-this extension may appear is `CH, EE, CEE`.
+`application_settings` with the value of 17613 (or 0x44cd); the list of messages
+in which this extension may appear is `CH, EE, CEE`.
+
+The ExtensionType value 17613 (or 0x44cd) is selected to replace the earlier
+experimental value 17513 (or 0x4469).
+
+In order to facilitate migration from the earlier experimental value, the server
+SHOULD support both code points 17613 (or 0x44cd) and 17513 (or 0x4469) during
+the transition period. Once the server receives a codepoint in the ClientHello
+message, the server MUST use the same codepoint to transmit server's settings.
+After completing the transition, the server SHOULD discontinue support for the
+old code point 17513 (0x4469). This ensures a smooth and interoperable migration
+to the newly assigned ExtensionType value, minimizing potential disruptions to
+the existing deployments.
```

Comment:
This guidance doesn't belong here. You should delete this parapgraph. "The transition" is not something a server can drive unilaterally. It describes a particular point in time that one particular client is running the process. The only reason the transition was viable at all was:

* Only one server needed to do the transition because, thus far, no one has been driving the standards side of ALPS
* ALPS has, thus far, only been used in a context where it was a performance optimization and can safely be disabled

Once you have multiple vendors, such transitions become more complicated. Avoiding that is, well, the point of standardization. Of course, sometimes things happen among early adopters, but those coordinations are transient and don't belong in a document like this. Indeed part of the requirement for early adoption of something is that it *not* become a critical dependency.

Reply:
ok, will delete the paragraph about this transition guidance.